deployment/docker-compose/docker-compose.yaml

```diff
@@ -10,21 +10,22 @@ services:
       MYSQL_USER: opik
       MYSQL_PASSWORD: opik
     healthcheck:
-      test: ["CMD", "mysqladmin", "ping", "-h", "127.0.0.1", "--silent"]
+      test: [ "CMD", "mysqladmin", "ping", "-h", "127.0.0.1", "--silent" ]
       timeout: 1s
       interval: 1s
       retries: 300
     ports:
-      - "3306:3306"
-    volumes: # Mounted on your $HOME folder
+      - "3306"
```

Comment:
This docker-compose file mostly aimed for local development. There's the use case of querying MySQL, ClickHouse and Redis as part of the development process. Also, there's the use case of directly curling the Opik backend service.

In my understanding with this change, we can still do it, but instead, they'll be exposed in random ports. Users will need to run an extra command to consult it.

We need to decided what we balance: risk of port conflict vs ease of use/development by exposing on standard ports. 

I'm inclined to the later.

Reply:
Discussed and agreed to offer an override.